.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -19,6 +19,6 @@ Expected:
 
 **Screenshots/terminal output (if relevant):**
 
-**flox Version (run `flox --version` if possible):**
+**Flox Version (run `flox --version` if possible):**
```

Comment:
nonblocking: in this case it's referring the version of the CLI, so my understanding was that we leave that lowercase

Reply:
The way I interpret the style guide, it is `flox` when referring to subcommands but `Flox` at all other times. So we might say "you should use the flox activate command" but would not say "flox Version: [version]".